jujugui/static/gui/src/app/store/env/bakery.js

```diff
@@ -301,14 +308,15 @@ YUI.add('juju-env-bakery', function(Y) {
     },
 
     /**
-      Interact to be able to sign-in to get authenticated.
-
-      @method _interact
-      @param {Function} The callback function to be sent in case of
-                        successful authentication
-      @param {Function} The callback function failure in case of
-                        wrong authentication
-    */
+     Interact to be able to sign-in to get authenticated.
+
+     @method _interact
+     @param {Function} The callback function to be sent in case of
+            successful authentication.
+     @param {Function} The callback function failure in case of
```

Comment:
This doc doesn't mention what the third parameter (actually the most important one) is.